MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -158,7 +164,12 @@ public sealed class CreditCard
         }
 
         /// <summary>
-        /// Whether or not the 3DS check was successful, as provided by the end
+        /// Whether or not the 3D-Secure verification (e.g. Safekey, SecureCode,
+        /// Verified by Visa) was successful, as provided by the end user.
+        /// `true` if customer verification was successful, or `false` if the
+        /// customer failed verification. If 3-D Secure verification was not
+        /// used, was unavailable, or resulted in an outcome other than success
+        /// or failure, do not include this successful, as provided by the end
```

Comment:
`do not include this successful...` seems like a copy+paste error

Reply:
💯